app/src/routes/api/v1/questionnaireRouter.js

```diff
@@ -125,7 +126,7 @@ function * checkPermission(next) {
 }
 
 function* checkAdmin(next) {
```

Comment:
Instead of changing the logic of the function why, why not stop invoking it? 
Now get a function called `checkAdmin` that check if a user is logged when already have been checked doesn't make sense for me